app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/attribute/form.phtml

```diff
@@ -21,13 +21,13 @@
     <input name="form_key" type="hidden" value="<?= $block->escapeHtml($block->getFormKey()) ?>" />
     <?= $block->getChildHtml('form') ?>
 </form>
-
-
-<script>
-require(['jquery', "mage/mage"], function(jQuery){
-
-    jQuery('#edit_form').mage('form').mage('validation', {validationUrl: '<?= /* @escapeNotVerified */ $block->getValidationUrl() ?>'});
-
-});
+<script type="text/x-magento-init">
+    {
+        '#edit_form': {
+            "Magento_Ui/catalog/product/edit/attribute": {
```

Comment:
File `catalog/product/edit/attribute.js` located in Magento_Catalog module instead of Magento_Ui